WeakAuras/Prototypes.lua

```diff
@@ -142,6 +142,45 @@ WeakAuras.function_strings = {
   ]]
 };
 
+local hsvFrame = CreateFrame("Colorselect")
```

Comment:
The RGB to HSV conversion isn't that complicated, I wonder wheter writing that method instead of using the Colorselect to transform would be easier.

Reply:
Considering that this is lifted almost directly from the url in the comment (with just a few modifications to adapt it to the WeakAuras environment), it certainly wouldn't be *easier* to write a home brew solution to do the same thing, though it'd probably be more transparent.